pkg/nanoid/nanoid.go

```diff
@@ -7,9 +7,11 @@ import (
 const (
 	defaultSize = 12
 
-	defaultCharacters = "_-0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	defaultCharacters = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 )
 
+type NanoID string
+
 func GetID() (string, error) {
```

Comment:
suggestion to return `NanoID , error` here and have `ToString()` method in `NanoID` type